packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/NotebookSessionController.java

```diff
@@ -153,6 +153,27 @@ ResponseEntity<NotebookSession> updateNotebookSession(
 		}
 	}
 
+	@PostMapping("/{id}/clone")
+	@Secured(Roles.USER)
+	@Operation(summary = "Clone a session")
+	@ApiResponses(value = {
+			@ApiResponse(responseCode = "200", description = "NotebookSession cloned.", content = @Content(mediaType = "application/json", schema = @io.swagger.v3.oas.annotations.media.Schema(implementation = NotebookSession.class))),
```

Comment:
Needs the swagger doc `@ApiResponse` for the 500 error case

Reply:
done